pyproject.toml

```diff
@@ -51,6 +51,7 @@ hydra-core = "^1.1.0"
 tqdm = "^4.50.2"
 websockets = "^10.1"
 pyyaml = "^5.4.0"
+detoxify = "^0.5.0"
```

Comment:
Just for organization, can you add this under a `# [tool.poetry.group.worker_addons.dependencies]` heading? I'll be toggling these in poetry 1.2.